### DISCUSSION TITLE
crash during mod generation

### DISCUSSION THREAD

**Discussion by Krupanjac** [jetelain/ArmaRealMap#175] [General]
If tool crashes for whatever reason while generating mod (This occurred while adding changes from 3den) , complete project becomes deprecated with no hopes of saving it.

It removes EVERYTHING from project including revision history and it's unsavable. This is really frustrating considering that 36km² map went to hell and now i can't make any changes to it.

**Comment by jetelain** [comment]
Hi,
Backup is probably still there. Can you contact me on Discord with more details on what went wrong using the error report tool from the help menu?
I think we can recover data.
Best regards,

Envoyé à partir de Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________
From: Arsen Dj. ***@***.***>
Sent: Thursday, November 2, 2023 3:41:06 PM
To: jetelain/ArmaRealMap ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [jetelain/ArmaRealMap] crash during mod generation (Discussion #175)


If tool crashes for whatever reason while generating mod (This occurred while adding changes from 3den) , complete project becomes deprecated with no hopes of saving it.

It removes EVERYTHING from project including revision history and it's unsavable. This is really frustrating considering that 36km² map went to hell and now i can't make any changes to it.

**Reply by Krupanjac** [reply]
I am on your discord but i already finished with new version on same map. This one had default assets which were not the best etc etc...

**Comment by jetelain** [comment]
Hi,

Have a look into this folder :
`%LOCALAPPDATA%\Packages\63155JulienEtelain.GameRealisticMapStudio_9mrjg6ss631xg\LocalCache\Local\GameRealisticMap\Arma3` or `%LOCALAPPDATA%\\GameRealisticMap\Arma3` (depending on which release you use)

You should find a folder `Backups` with a sub folder having the name of the map. Within you will have multiple zip files. Take the most recent, and extract it into your project drive. It should repair the map.

If you need, we could have a call/screen sharing on Discord ( https://discord.gg/9JTHHKWJPn )

I think binarize have failed, and might have left some files in an invalid state, preventing tool to reopen the file.

Best regards,
